src/af/validation.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Validation Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines contain the default error messages used by
+    | the validator class. Some of these rules have multiple versions such
+    | as the size rules. Feel free to tweak each of these messages here.
+    |
+    */
+
+    'accepted' => 'Die :attribute moet aanvaar word.',
+    'active_url' => 'Die :attribute is nie \'n geldig URL.',
```

Comment:
Use " when you have ' in translations

For example here :  'active_url' => "Die :attribute is nie 'n geldig URL.",